client-http/src/test/scala/com/cloudera/livy/client/http/HttpClientSpec.scala

```diff
@@ -18,11 +18,15 @@
 
 package com.cloudera.livy.client.http
 
+import java.io.{ByteArrayInputStream, File}
 import java.net.{InetAddress, URI}
+import java.nio.file.{Paths, Files}
```

Comment:
Hmm, so this change is still there - but the style checker seems to miss it (it passes locally too for me). Anyway, fixed it now.